src/lib/expression/evaluation/expression_evaluator.cpp

```diff
@@ -1298,29 +1327,35 @@ std::shared_ptr<ExpressionResult<Result>> ExpressionEvaluator::_evaluate_binary_
 template <typename Functor>
 void ExpressionEvaluator::_resolve_to_expression_result_view(const AbstractExpression& expression,
                                                              const Functor& functor) {
-  _resolve_to_expression_result(expression,
-                                [&](const auto& result) { result.as_view([&](const auto& view) { functor(view); }); });
+  _resolve_to_expression_result(expression, [&](const auto& result) {
+    result.as_view([&](const auto& view) {
+      functor(view);
+    });
+  });
 }
 
 template <typename Functor>
 void ExpressionEvaluator::_resolve_to_expression_result_views(const AbstractExpression& left_expression,
                                                               const AbstractExpression& right_expression,
                                                               const Functor& functor) {
-  _resolve_to_expression_results(
-      left_expression, right_expression, [&](const auto& left_result, const auto& right_result) {
-        left_result.as_view([&](const auto& left_view) {
-          right_result.as_view([&](const auto& right_view) { functor(left_view, right_view); });
-        });
-      });
+  _resolve_to_expression_results(left_expression, right_expression,
+                                 [&](const auto& left_result, const auto& right_result) {
+                                   left_result.as_view([&](const auto& left_view) {
+                                     right_result.as_view([&](const auto& right_view) {
+                                       functor(left_view, right_view);
+                                     });
+                                   });
+                                 });
```

Comment:
pure beauty ...

Reply:
That's actually a case where I think it's helpful. Because there is a stacking that is otherwise not obvious. Depending on the view type, these are nested loops.

Reply:
absolutely, it's just the whitespace that makes it look interesting. And it's not really a nested loop, right? That depends completely on what the functor does with the two results (and the only code location that calls this method iterates both simultaneously).

Reply:
True, it might not be a nested loop. I just skimmed over the code and saw that those views can be large. Haven't looked what the functors eventually do.

Reply:
Well, these views can as well also just wrap one single valie. Anyway, that's nothing that matters for this code block at all, the formatting is apparently correct, feel free to resolve.